### PR TITLE
Custom spacing for input toolbar items

### PR DESCRIPTION
**Pull request by kohtenko**

### CODE REVIEW

**Comment by jessesquires** [comment]
thanks @kohtenko !

File: JSQMessagesViewController/Views/JSQMessagesToolbarContentView.h [hunk 1]
```diff
@@ -59,6 +59,13 @@ FOUNDATION_EXPORT const CGFloat kJSQMessagesToolbarContentViewHorizontalSpacingD
 @property (assign, nonatomic) CGFloat leftBarButtonItemWidth;
 
 /**
+ *  Specifies the left constant of the `Leading` constraint of `leftBarButtonItem`
```
Comment (jessesquires) [inline JSQMessagesViewController/Views/JSQMessagesToolbarContentView.h]: Let's change to:

> Specifies the amount of spacing between the content view and the leading edge of `leftBarButtonItem`.

File: JSQMessagesViewController/Views/JSQMessagesToolbarContentView.h [hunk 2]
```diff
@@ -59,6 +59,13 @@ FOUNDATION_EXPORT const CGFloat kJSQMessagesToolbarContentViewHorizontalSpacingD
...
+ *
+ * @discussion Default value is 8.0f
```
Comment (jessesquires) [inline JSQMessagesViewController/Views/JSQMessagesToolbarContentView.h]: 2 spaces after the `*`

Let's change to:
> *  `@discussion` The default value is `8.0f`

File: JSQMessagesViewController/Views/JSQMessagesToolbarContentView.h [hunk 3]
```diff
@@ -59,6 +59,13 @@ FOUNDATION_EXPORT const CGFloat kJSQMessagesToolbarContentViewHorizontalSpacingD
...
+ */
+@property (assign, nonatomic) CGFloat leftBarButtonSpacing;
```
Comment (jessesquires) [inline JSQMessagesViewController/Views/JSQMessagesToolbarContentView.h]: let's rename `leftContentPadding`

File: JSQMessagesViewController/Views/JSQMessagesToolbarContentView.h [hunk 4]
```diff
@@ -88,6 +95,13 @@ FOUNDATION_EXPORT const CGFloat kJSQMessagesToolbarContentViewHorizontalSpacingD
 @property (assign, nonatomic) CGFloat rightBarButtonItemWidth;
 
 /**
+ *  Specifies the right constant of the `Trailing` constraint of `rightBarButtonItem`
+ *
+ * @discussion Default value is 8.0f
+ */
+@property (assign, nonatomic) CGFloat rightBarButtonSpacing;
```
Comment (jessesquires) [inline JSQMessagesViewController/Views/JSQMessagesToolbarContentView.h]: Same comments here as above, just swapping left/right and leading/trailing.

**Comment by jessesquires** [comment]
Hey @kohtenko - Left a few comments for you. Address those, then I'll be happy to merge. :tada: :smile:

**Comment by kohtenko** [comment]
@jessesquires Unfortunately Travis fails for some reason. Local tests went without failures

**Comment by jessesquires** [comment]
Thanks @kohtenko ! :+1:

**Comment by jessesquires** [comment]
thanks again @kohtenko !

img

**Pull request merged**